docs/how-to/extending/sql.qmd

```diff
@@ -0,0 +1,186 @@
+# Working with raw SQL strings
+
+While Ibis is all about avoiding raw SQL strings, there are few reasons you
+might need to use them:
```

Comment:
Not sure if I'd want to say Ibis is "all about avoiding raw SQL". How about:

```suggestion
While Ibis generally lets you avoid writing SQL, there are a few use cases where
you might  want to intermingle SQL and Ibis:
```

Reply:
I am trying to set the stage for the remaining content.

I want the reader to be thinking about raw SQL strings, not juggling general versus specific notions of what ibis can or cannot do.

I don't think we need to use language that preserves our right to state "avoiding raw SQL isn't the only thing ibis does".

Reply:
Reworded this. LMK what you think!